noxfile.py

```diff
@@ -206,16 +190,30 @@ def make(session: nox.Session):
     """
     Generate HTML from documentation source using the Makefile
     """
-    parser = _relaxed_parser(session)
-    parser.add_argument(
-        "make_args", nargs="*", help="Specify make targets as arguments"
-    )
-    args = parser.parse_args(session.posargs)
+    make_args = session.posargs or ["clean", "coredocs"]
 
-    install(session, req="requirements-relaxed" if args.relaxed else "requirements")
+    install(session, req="requirements")
     _clone_core_check(session)
-    make_args: list[str] = [
+    session.run(
+        "make",
+        "-C",
+        "docs/docsite",
         f"PYTHON={_env_python(session)}",
-        *(args.make_args or ("clean", "coredocs")),
-    ]
-    session.run("make", "-C", "docs/docsite", *make_args, external=True)
+        *make_args,
+        external=True,
+    )
+
+
+@nox.session
+def tag(session: nox.Session):
```

Comment:
Same as https://github.com/ansible/ansible-documentation/pull/2362#discussion_r1920519239, `tag` shouldn't be there.

Reply:
fixed. thanks for catching!